### PR TITLE
ADR-244 Extended Content Addressable URNs for Decentraland wearables

### PR DESCRIPTION
**Pull request by aleortega**
This PR introduces ADR-244, outlining the necessary changes for implementing the Extended Wearables Addressable feature. This enhancement aims to ensure the unique identification of each minted wearable item by extending their URNs to include the tokenId property as the last part.

In the upcoming iteration of this ADR, we will provide a list of endpoints to be modified, along with a comprehensive explanation of the applied changes.

### CODE REVIEW

File: content/ADR-244-extended-wearables-addressable.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+decentraland:{protocol}:collections-v1:{contract(0x[a-fA-F0-9]+)}:{itemId}:{tokenId}
+```
+
+* **tokenID**: The unique identifier of the minted wearable NFT item, distinguishing it from other items of the same wearable content type and content identifier.
+
+The token ID part is optional and will only be used when addressing specific minted wearable NFT items. For other content types, such as scenes or other types of NFTs, the token ID part will not be added at the time of this writing.
+
+The migration of the minted wearables to the new URN format will be done in a progressive manner. The **Explorer** will show the items grouped by item kinds but allow the user to select the specific NFT to add to their profile. Once the users save their profile for the first time once this ADR is running effectively, a process in the client should select the first NFT for each item kind saved in the profile.
```
Comment (marianogoldman) [inline content/ADR-244-extended-wearables-addressable.md:51]:
```suggestion
The migration of the minted wearables to the new URN format will be done in a progressive manner. The **Explorer** will show the items grouped by item kinds but allow the user to select the specific NFT to add to their profile. Once the users save their profile for the first time and this ADR is running effectively, a process in the client should select the first NFT for each item kind saved in the profile.
```

File: content/ADR-244-extended-wearables-addressable.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The extended content-addressable URN format for Decentraland wearables will be applied over collections v1 and v2 specifically. This extension was already proposed in the [ADR-109](/adr/ADR-109) on the second alternative:
+
+```
+decentraland:{protocol}:collections-v1:{contract(0x[a-fA-F0-9]+)}:{itemId}:{tokenId}
+```
+
+* **tokenID**: The unique identifier of the minted wearable NFT item, distinguishing it from other items of the same wearable content type and content identifier.
+
+The token ID part is optional and will only be used when addressing specific minted wearable NFT items. For other content types, such as scenes or other types of NFTs, the token ID part will not be added at the time of this writing.
```
Comment (marianogoldman) [inline content/ADR-244-extended-wearables-addressable.md:49]: What about emotes? Should NFT emotes follow the same approach? In any case, scenes don't use urns for pointers. I think currently urns are only for wearables, emotes and stores.

File: content/ADR-244-extended-wearables-addressable.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+2. **Enhanced Ownership Validation**: The extended URN format significantly enhances the ownership validation process for minted wearables. By leveraging the unique token ID, the Catalyst's content validation system can effectively verify and confirm ownership of individual minted wearable items, streamlining interactions within the Decentraland metaverse.
+
+Let's consider an example where a user mints two identical blue shirts from a specific V2 collection. Currently, the URNs for each NFT appear as follows:
+* **First blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a
+* **Second blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a
+
+Once this ADR takes effect, the URNs will be modified as follows:
+* **First blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a:1
```
Comment (marianogoldman) [inline content/ADR-244-extended-wearables-addressable.md:34]: I thought the last part was going to be a long, globally unique value. This looks like a local id within the collection. Let's check with @hugoArregui if this is enough for on-chain validation.

File: content/ADR-244-extended-wearables-addressable.md (hunk 2 above)
Comment (aleortega) [reply content/ADR-244-extended-wearables-addressable.md:49]: Added emotes to the ADR as discussed.

**Review by marianogoldman — APPROVED** [review]

**Pull request merged**